### ISSUE TITLE
Incorrect deleted status for asset holding

### ISSUE THREAD

**Issue by jasonpaulos** [algorand/indexer#459]
## Summary
The API call `/v2/accounts/FLUFMFGK4DTSTU4JROQQOJZXIBMALAVS2XEEQXLFYZRSUEVVQ5HSQXGM4Q` on a MainNet Indexer 2.3.3 yields the following:

```
{
  "account": {
    "address": "FLUFMFGK4DTSTU4JROQQOJZXIBMALAVS2XEEQXLFYZRSUEVVQ5HSQXGM4Q",
    ...
    "assets": [
      ...
      {
        "amount": 0,
        "asset-id": 144871850,
        "creator": "",
        "deleted": false,
        "is-frozen": false,
        "opted-in-at-round": 12421358,
        "opted-out-at-round": 12424246
      },
      ...
    ],
    ...
}
```

The asset 144871850 was created by `FLUFMFGK4DTSTU4JROQQOJZXIBMALAVS2XEEQXLFYZRSUEVVQ5HSQXGM4Q` and subsequently deleted, so I would expected deleted to be true (and for the asset to not show up at all unless `include-all` is enabled).

I suspect there's something special about this specific account/asset, since the account `WSJHNPJ6YCLX5K4GUMQ4ISPK3ABMS3AL3F6CSVQTCUI5F4I65PWEMCWT3M` on TestNet also has an asset that is created and deleted, but indexer correctly reports its status.

## Scope/Requirements
This should be investigated further.

## Urgency/Relative Priority
Indexer is reporting an incorrect value.

**Comment by ian-algorand** [comment]
@algojohnlee can we check this out if the newest indexer release (2.3.4) fixes this? We can close if so.

**Comment by onetechnical** [comment]
Yes, this will be fixed with the 2.3.4 release.
